offer-manager-domain/src/main/groovy/life/qbic/business/offers/content/CreateOfferContent.groovy

```diff
@@ -123,7 +123,7 @@ class CreateOfferContent implements CreateOfferContentInput, FetchOfferOutput{
             return 0
         } else {
             return offerItems.sum {
-                it.quantity * it.unitPrice
+                it.quantity * it.unitPrice - it.quantityDiscount
```

Comment:
we have the total net price already computed, so no need to calculate the net item here again

Reply:
We need to calculate them for the specific groups. We only have them for all items.